test/global.test.ts

```diff
@@ -5,12 +5,17 @@
 
 import { IHookCallbackContext } from 'mocha';
 import * as vscode from 'vscode';
+import { TestOutputChannel } from 'vscode-azureextensiondev';
+
+import { ext } from '../extension.bundle';
 
 // tslint:disable-next-line:strict-boolean-expressions export-name
 export let longRunningTestsEnabled: boolean = !/^(false|0)?$/i.test(process.env.ENABLE_LONG_RUNNING_TESTS || '');
+export let deleteDefaultResourceGroup: boolean = !/^(false|0)?$/i.test(process.env.DELETE_DEFAULT_RESOURCE_GROUP || '');
```

Comment:
I think you need to flip the name/logic of this flag so that it's on by default. I really want to make sure these tests aren't leaving around azure resources.